IN-global_conf.json

```diff
@@ -4,75 +4,76 @@
 		"clksrc": 1,
 		"clksrc_desc": "radio_1 provides clock to concentrator for most devices except MultiTech. For MultiTech set to 0.",
 		"antenna_gain": 0,
```

Comment:
Vanilla version is prepared for testing by Velmurugan of TTN Chennai / MCCI. I expect an answer overnight, depending on his workload.